crates/utils/re_log/src/setup.rs

```diff
@@ -39,7 +39,8 @@ pub fn setup_logging() {
                 use std::io::Write as _;
                 writeln!(
                     buf,
-                    "{}:{} {}",
+                    "{} {}:{} {}",
+                    record.level(),
```

Comment:
what is this? I'm lacking context

Reply:
![context](https://github.com/user-attachments/assets/b999f64f-5a69-46da-a490-bffa08ce5ddc)

Reply:
I like to flip that switch so I can see where a log message comes from.
Before I couldn't see the log level when I did so. Now I can.

![image](https://github.com/user-attachments/assets/e452730d-1850-4684-8396-cbfa65d371df)

Reply:
It was completely unrelated to the PR, except I wanted to debug "what the hell is wring that annoying log message"